### PR TITLE
Changed the title and Navbar color

### PR DESCRIPTION
**Pull request by hrushikesh-git**
Chaneged the Title of the blog and the color of the Navbar as instructed, 
navbar change

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -41,6 +41,12 @@ const blogSchema = {
     type: Date,
     default: Date.now
   },
+  likes:[
+    {
+      type: mongoose.Schema.Types.ObjectId,
+       ref: "User"
+    }
+  ],
```
Comment (Pranav016) [inline app.js:49]: why was this added in a PR made for improving navbar ?

File: app.js [hunk 2]
```diff
@@ -49,6 +55,8 @@ const blogSchema = {
 
 //Making a MongoDB model for the schema-
 const Blog = new mongoose.model("Blog", blogSchema);
+mongoose.connect(URL, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.set('useCreateIndex', true);
```
Comment (Pranav016) [inline app.js:59]: These additions were not required, they were already present in `app.js`
image

**Review by Pranav016 — CHANGES_REQUESTED** [review]
You cannot make a PR from master to master, make from a new branch to develop. Don't commit to master..

**Comment by Pranav016** [comment]
Make a PR from new branch, closing this. reopen if required

File: app.js (hunk 1 above)
Comment (hrushikesh-git) [reply app.js:49]: I think this was already present in my code that why it got uploaded and  anyways I will make the requested changed and will get back

Comment (Pranav016) [reply app.js:49]: No problem 😊
